Gems/EMotionFX/Code/Source/Editor/SkeletonModel.cpp

```diff
@@ -413,7 +444,7 @@ namespace EMotionFX
         case ROLE_HITDETECTION:
         {
             const AZStd::shared_ptr<PhysicsSetup>& physicsSetup = m_actor->GetPhysicsSetup();
-            if (physicsSetup)
+            if (!isRootNode && physicsSetup)
```

Comment:
question: As `!isRootNode` appears in all these conditions could we instead possibly have it as an early return instead? This might be awkward to do, just wanted to check